setup.py

```diff
@@ -32,9 +32,9 @@
         'nbgrader.exporters'
     ],
     package_data={
-        '': [
-            'nbextensions/*.js',
-            'nbextensions/*.css'
+        'nbgrader': [
```

Comment:
It's on my todo list. I didn't acchieve to access it from with the python module while it was outside of `/nbgrader`. But now the it works, I can tackle that again.